src/main/java/hw3/ex2/ElementsPageFluent.java

```diff
@@ -0,0 +1,75 @@
+package hw3.ex2;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ElementsPageFluent {
+
+    private WebDriver driver;
+
+    public ElementsPageFluent(WebDriver driver){
+        this.driver = driver;
+        PageFactory.initElements(driver,this);
+    }
+
+    @FindBy(xpath = "//label[normalize-space()='Water']")
+    private WebElement checkBoxWater;
+
+    @FindBy(xpath = "//label[normalize-space()='Wind']")
+    private WebElement checkBoxWind;
+
+    @FindBy(xpath = "//label[normalize-space()='Selen']" )
+    private WebElement radioSelen;
+
+    @FindBy(css = "div.colors .uui-form-element")
+    private WebElement colorsDropDown;
+
+    @FindBy(xpath = "//*[text()='Yellow']")
+    private WebElement yellow;
```

Comment:
1. checkbox and radiobutton should be definied as `List<WebElement>`
2. Why do you do not use `Select` for the dropdown?